Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
+
+**Warning:** Even if you don't see an Emax, that doesn't mean there isn't one. Be very careful about using linear models for Dose-Response relationships. Extrapolation outside of the observed dose range could indicate a higher dose is always better (even if it isn't).
+
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE} 
+pd_data$DAY_label <- paste("Day", pd_data$PROFDAY)
+pd_data$DAY_label[pd_data$DAY_label=="Day 0"] = "Baseline"
+pd_data_to_plot <- pd_data[pd_data$DAY_label%in%c("Baseline","Day 6"),]
+
+
+gg <- ggplot(data = pd_data_to_plot, aes(x=DOSE,y=LIDV, group=DOSE))
+gg <- gg + xgx_stat_ci(conf_level = .95)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + facet_grid(~DAY_label)
+gg <- gg + labs(x=dose_label, y=pd_label)
+gg <- gg + xgx_annotate_status(status)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+### PD marker by Dose, faceted by visit, mean (95% CI) by Dose
+
+Sometimes the crossectional Dose-Response curve which looks only at one timepoint defined in the protocol can obscure certain characteristics of the dose-response relationship. For example, if the response variable is much delayed compared to PK the maximal PD effect could occur much later than steady state PK is achieved. Looking only at the defined clinical endpoint has the potential miss this, especially in early clinical trials before the time course of the effect has been characterized. Looking at longitudinal PD over time (as in previous sections above) can help to uncover these trends. It may also be helpful to plot the cross-sectional Dose-Response curves for different time points throughout the study.
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE} 
+
+pd_data$DAY_label <- paste("Day", pd_data$PROFDAY)
+pd_data$DAY_label[pd_data$DAY_label=="Day 0"] = "Baseline"
+pd_data_to_plot <- pd_data[pd_data$DAY_label%in%c("Baseline","Day 2", "Day 4", "Day 6"),]
+
+
+gg <- ggplot(data = pd_data_to_plot, aes(x=DOSE,y=LIDV, group = DOSE))
+gg <- gg + xgx_stat_ci(conf_level = .95)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + facet_grid(~DAY_label)
+gg <- gg + labs(x=dose_label, y=pd_label)
+gg <- gg + xgx_annotate_status(status)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+### Explore covariate effects on Dose-Response relationship
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE} 
+
+pd_data$DAY_label <- paste("Day", pd_data$PROFDAY)

```

Comment:
same comments as above re: DAY_label and use of %>%

Reply:
ok changed now